src/Hooks/FiltersTracking.md

```diff
@@ -73,3 +73,25 @@ public function getIntegrationHubspotLocalStorageMap(array $params, array $stora
 	return $params;
 }
 ```
+
+## Change amount of days localStorage is storing the data.
+We store localStorage for 30 days by default, with this filter you can change this value.
+
+**Filter name:**
+`es_forms_tracking_expiration`
+
+**Filter example:**
+```php
+// Get tracking expiration days.
+\add_filter('es_forms_tracking_expiration', [$this, 'getTrackingExpiration']);
+
+/**
+ * Get tracking expiration days.
+ *
+ * @return string
+ */
+public function getTrackingExpiration(): string
```

Comment:
why not add a parameter here so when calling a filter it can be any value and not fixed?

Reply:
no sure what you are getting at, can you please provide little more context